src/group_theory/perm/sign.lean

```diff
@@ -761,3 +765,43 @@ end congr
 end sign
 
 end equiv.perm
+
+section alternating_subgroup
+open equiv.perm
+variables (α) [fintype α] [decidable_eq α]
+
+/-- The alternating group on a finite type, realized as a subgroup of `equiv.perm`.
+  For $A_n$, use `alternating_subgroup (fin n)`. -/
+@[derive fintype] def alternating_subgroup : subgroup (perm α) :=
+sign.ker
+
+instance [subsingleton α] : unique (alternating_subgroup α) :=
+⟨⟨1⟩, λ ⟨p, hp⟩, subtype.eq (subsingleton.elim p _)⟩
+
+variables {α}
+
+lemma alternating_subgroup_eq_sign_ker : alternating_subgroup α = sign.ker := rfl
+
+@[simp]
+lemma mem_alternating_subgroup {f : perm α} :
+  f ∈ alternating_subgroup α ↔ sign f = 1 :=
+sign.mem_ker
+
+lemma two_mul_card_alternating_subgroup [nontrivial α] :
+  2 * card (alternating_subgroup α) = card (perm α) :=
+begin
+  classical,
+  rw ← fintype.card_units_int,
+  convert (card_eq_card_quotient_mul_card_subgroup (alternating_subgroup α)).symm,
+  convert of_equiv_card (quotient_group.quotient_ker_equiv_of_surjective _
+    (sign_surjective α)).to_equiv,
+end
+
+@[simp]
+lemma card_alternating_subgroup_eq_one [h : subsingleton α] :
+  card (alternating_subgroup α) = 1 :=
+by convert card_of_subsingleton (1 : alternating_subgroup α)
```

Comment:
After #7008, this is true `by simp`.

Reply:
Should I merge that PR into this one and wait, or add that change to #7008?

Reply:
I suppose you could just remove this lemma, since the linter will reject `@[simp]` once that PR is merged, and `alternating_group.unique` states the same thing anyway?

Reply:
I just removed it, you're right.